test/custom-parser.3.test.js

```diff
@@ -237,9 +237,10 @@ test('catch all content type parser should not interfere with content type parse
         'Content-Type': 'text/html'
       }
     }, (err, response, body) => {
-      t.error(err)
-      t.equal(response.statusCode, 200)
-      t.same(body.toString(), 'my texthtml')
+      t.assert.ifError(err)
+      t.assert.strictEqual(response.statusCode, 200)
+      t.assert.deepStrictEqual(body.toString(), 'my texthtml')
+      done()
```

Comment:
When we run multiple calls in parallel (no await), we must wait for them all to be completed before calling `done`.

In this scenario, the last sget may end before the other 2, and the test would complete anyway.

Example solution: https://github.com/fastify/fastify/pull/5881/files#diff-6a98dbdd7ca9fb8e0adfcc8787c950f65344cd9581437118754f4adb1a791f7d

Reply:
If I understand correctly, I have to add this:
```ts
      let pending = 2
      function completed () {
        if (--pending == 0) {
          done()
        }
      }
```
correct ?

Reply:
In parallel, that would work